app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_create :create_remember_token
+  has_secure_password
```

Comment:
The Post model has the `belongs_to :user` association in it. The User model therefore should reflect it by having `has_many :posts` counterpart